cwltool/singularity.py

```diff
@@ -47,7 +47,7 @@ def _singularity_supports_userns() -> bool:
                 stdout=DEVNULL,
                 universal_newlines=True,
             ).communicate(timeout=60)[1]
-            _USERNS = "No valid /bin/sh" in result
+            _USERNS = "/bin/sh doesn't exist in container" in result
```

Comment:
Can we get a test for this?

Reply:
Thanks for the test. In which version of Singularity did the error message change?

We should support both error messages.

Reply:
@mr-c I see that I'm currently failing the Travis build test because of permission denials for the test I added. Do I need to edit this? I tested it on my local machine and it seemed to be working fine.